tests/integration/hive_metastore/test_pipeline_migrate.py

```diff
@@ -0,0 +1,91 @@
+from databricks.sdk.service.pipelines import PipelineLibrary, NotebookLibrary
+
+from databricks.labs.ucx.assessment.pipelines import PipelinesCrawler
+from databricks.labs.ucx.hive_metastore.pipelines_migrate import PipelinesMigrator, PipelineRule, PipelineMapping
+from ..assessment.test_assessment import _PIPELINE_CONF
+
+_TEST_STORAGE_ACCOUNT = "storage_acct_1"
+_TEST_TENANT_ID = "directory_12345"
+
+
+# pylint: disable=too-many-locals
+def test_pipeline_migrate(
+    ws,
+    make_pipeline,
+    make_notebook,
+    make_random,
+    watchdog_purge_suffix,
+    make_directory,
+    inventory_schema,
+    sql_backend,
+    runtime_ctx,
+):
+    src_schema = runtime_ctx.make_schema(catalog_name="hive_metastore")
+    target_schemas = 5 * [runtime_ctx.make_schema(catalog_name="hive_metastore")]
+
+    dlt_notebook_path = f"{make_directory()}/dlt_notebook.py"
+    src_table = runtime_ctx.make_table(catalog_name="hive_metastore", schema_name=src_schema.name, non_delta=True)
+    dlt_notebook_text = (
+        f"""create streaming table st1\nas select * from stream(hive_metastore.{src_schema.name}.{src_table.name})"""
+    )
+    make_notebook(content=dlt_notebook_text.encode("ASCII"), path=dlt_notebook_path)
+
+    pipeline_names = []
+    created_pipelines = []
+    for _ in range(3):
```

Comment:
create just one pipeline, otherwise the tests become too flaky after this is merged.

Reply:
Done! Although I think we should be testing all possible combinations. Agree with flakiness.